web/ui/dashboard/src/app/private/components/add-analytics/add-analytics.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="padding-x__30px padding-y__40px">
+	<h3>Welcome, {{ authDetails?.first_name }} 🌤</h3>
+	
+	<form [formGroup]="addAnalyticsForm" (ngSubmit)="addAnalytics()">
+		<h3 class="margin-top__24px">
+			Would you like to enable analytics?
+		</h3>
+		<p class="font__14px color__grey margin-bottom__20px width__80">Enabling Analytics would allow anonymous metrics to be taken to see how Convoy is being used.</p>
```

Comment:
Please update the text here to: Enabling Analytics will provide anonymous metrics to help us understand how you use Convoy, so we can make it even better for you.